docs/faq.rst

```diff
@@ -0,0 +1,26 @@
+Frequently Asked Questions
+==========================
+
+Why is Curio not supported as a backend?
+----------------------------------------
+
+Curio_ was supported in AnyIO before v3.0. Support for it was dropped for two reasons:
+
+#. Its interface allowed only coroutine functions to access the Curio_ kernel. This forced AnyIO to
+   follow suit in its own API design, making it difficult to adapt existing applications that
+   relied on synchronous callbacks to us AnyIO. It also interfered with the goal of matching Trio's
```

Comment:
to *use*

Reply:
Good catch. My "e" key has been working a bit erratically as of late.